pkg/api/handlers/compat/images_build.go

```diff
@@ -156,7 +164,15 @@ func BuildImage(w http.ResponseWriter, r *http.Request) {
 		output = query.Tag[0]
 	}
 	format := buildah.Dockerv2ImageManifest
+	registry := query.Registry
+	isolation := buildah.IsolationChroot
+	/*
+		// FIXME, This is very broken.  Buildah will only work with chroot
+		isolation := buildah.IsolationDefault
+	*/
 	if utils.IsLibpodRequest(r) {
+		//		isolation = buildah.Isolation(query.Isolation)
```

Comment:
should we drop this commented code?

Reply:
Lets merge this.  I waiting to open a PR to fix this as soon as this PR merges.